pkgs/data/themes/qogir/default.nix

```
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, gdk_pixbuf, librsvg, gtk-engine-murrine }:
+
+stdenv.mkDerivation rec {
+  name = "qogir-${version}";
+  version = "2018-11-12";
+
+  src = fetchFromGitHub {
+    owner = "vinceliuice";
+    repo = "qogir-theme";
```

```suggestion
    repo = pname;
```
https://github.com/NixOS/nixpkgs/pull/52918/files#r244081530

---

Done.